pelican/plugins/image_process/image_process.py

```diff
@@ -328,8 +328,7 @@ def harvest_images_in_fragment(fragment, settings):
 
         elif not isinstance(d, dict):
             raise TypeError(
-                f"Derivative {derivative} definition not handled"
-                "(must be list or dict)"
+                f"Derivative {derivative} definition not handled(must be list or dict)"
```

Comment:
```suggestion
                f"Derivative {derivative} definition not handled (must be list or dict)"
```
This would fix issue #75.

Reply:
Space added. Thank you!